velox/exec/fuzzer/CacheFuzzer.cpp

```diff
@@ -211,6 +225,25 @@ void CacheFuzzer::initSourceDataFiles() {
       fileSizes_.emplace_back(fileSize);
     }
   }
+
+  if (FLAGS_enable_file_faulty_injection) {
+    faultyFileSystem()->setFileInjectionHook([&](FaultFileOperation* op) {
+      if (std::regex_match(op->path, kDataFileNamePattern)) {
+        // Skip errors on input data files.
+        return;
+      }
+      std::random_device rd;
+      boost::random::uniform_int_distribution<int> dist(1, 100);
+      if (op->type == FaultFileOperation::Type::kWrite &&
+          dist(rd) <= kFileWriteErrorRate) {
+        VELOX_FAIL("Inject hook write failure");
+      }
+      if (op->type == FaultFileOperation::Type::kReadv &&
```

Comment:
ssd cache all use readv? Thanks!

Reply:
actually evictlog uses read. let me add that case to the injection.